lib/fizzy/execute.cpp

```diff
@@ -753,4 +753,13 @@ execution_result execute(const Module& module, FuncIdx function, std::vector<uin
     auto instance = instantiate(module);
     return execute(instance, function, args);
 }
+
+std::optional<FuncIdx> find_exported_function(const Module& module, std::string_view name)
+{
+    for (const auto& export_ : module.exportsec)
```

Comment:
1. I suggest following the coding rule to enclose a body of `for`, `if,` etc. with `{}` if body has more than 2 lines of code. This is enforceable by clang-tidy.
2. You can use `std::find_if`.